myenumerable.rb

```diff
@@ -0,0 +1,17 @@
+module Myenumerable
+  def all?(&block)
+    each { |n| return false unless block.call(n) }
+    true
+  end
+
+  def any?(&block)
+    each { |n| return true if block.call(n) }
+    false
+  end
+
+  def filter(&block)
+    result = []
+    each { |n| result.push(n) if block.call(n) }
+    result
+  end
+end
```

Comment:
- [Optional] Good job Team, my suggestion here will be to have some validation to make sure the method is called with block, something like ``return to_enum(:each) unless block_given?``